### ISSUE TITLE
npm deprecation on tsc package

### ISSUE THREAD

**Issue by dutronlabs** [Microsoft/vsts-task-lib#220]
Following the vsts-task-lib node installation instructions, it appears tsc is deprecated and typescript should be used. Is this the case?

```
administrators-MacBook-Pro:node administrator$ sudo npm install tsc -g
Password:
npm WARN deprecated tsc@1.20150623.0: You probably meant to instally 'typescript'. Run 'npm install typescript -g'
/usr/local/bin/tsc -> /usr/local/lib/node_modules/tsc/bin/tsc
/usr/local/bin/tsserver -> /usr/local/lib/node_modules/tsc/bin/tsserver
/usr/local/lib
└── tsc@1.20150623.0 
```

**Comment by bryanmacfarlane** [comment]
This is no longer in the docs.  We can re-open if I missed something ...

**Comment by ZelphirKaltstahl** [comment]
> Is this the case?

(I vote for re-opening this issue.)

**Comment by stephenmichaelf** [comment]
@ZelphirKaltstahl Can you help us understand the issue you are running in to?

**Comment by ZelphirKaltstahl** [comment]
It seems there is not much context in the answer that closes the issue. The issue asked whether it is the case that one should install typescript with a changed command, but that question was not really answered. It is not an issue of functionality any more, but rather how clear the answer is.

> This is no longer in the docs. We can re-open if I missed something ...

This is confusing, because:

* Which docs? Where can I find them? When I wrote, I had no clue, that "docs" could relate to the readme file of the repository. I only discovered that later. Usually when I read "docs" I think of something comprehensive, a whole website like some "readthedocs" page or a comprehensive tutorial, but I would not expect it to be a simple readme file.
* It is also ambiguous, because "What is not in the docs any more?" (a) The information that one should use another command, or (b) the information, that one should use the old command.

It is really not a big issue, but I found the closing answer to be unclear and it might confuse others too. 

At least a link to the "docs" would have cleared things up or a phrase like "docs in the readme of this repo". I was only missing some context when reading this. Now I know the answer : )

The command to install the typescript compiler changed and one should really use a new command, mentioned in the output on the command line.

**Comment by bryanmacfarlane** [comment]
@ZelphirKaltstahl - right on the front page there's documentation link for build node lib to this

https://github.com/Microsoft/vsts-task-lib/blob/master/node/README.md

It used to have instructions to install tsc globally (tsc -g referenced above).  It was changed awhile ago to do an `npm install` with typescript as a dependency.  As I stated when closing the issue, the step he was having an issue with is no longer in the docs.

I hope that clarifies it.
